client/src/charts/canada/CanadaD3Component.js

```diff
@@ -153,14 +150,11 @@ export class CanadaD3Component {
     };
 
     // Set province colours, attach event dispatchers
-    const province_is_active = (prov_key) =>
-      _.some(data, (year) => year[prov_key]);
+    const province_is_active = (prov_key) => data[prov_key];
```

Comment:
Not sure if you want to change it back to how it was before but

![image](https://user-images.githubusercontent.com/25855114/90544706-864ec380-e155-11ea-9299-6a8f77927d64.png)

Previously, the provinces/territories with historical data still had colour showing you could hover over it.

If you choose to keep it this way, you could remove the method altogether like this
`const get_color = (prov_key) =>
      data[prov_key] ? main_color : secondary_color;
    const get_opacity = (prov_key) =>
      data[prov_key] ? color_scale(data[prov_key] || 0) : 0.5;`